x-pack/test/alerting_api_integration/spaces_only/tests/alerting/execution_status.ts

```diff
@@ -183,7 +183,7 @@ export default function executionStatusAlertTests({ getService }: FtrProviderCon
       await ensureAlertUpdatedAtHasNotChanged(alertId, alertUpdatedAt);
     });
 
-    it('should eventually have error reason "unknown" when appropriate', async () => {
+    it('should eventually have error reason "validate" when appropriate', async () => {
```

Comment:
As mentioned ^^^, previously validation errors returned a reason of `unknown`, but now return the new reason.  I didn't see an obvious way to test the `unknown` status anymore, which I guess is kinda good!